publish/working-with-pipelines/working-with-pipelines.md

```diff
@@ -402,7 +429,11 @@ Follow these steps:
 
    Alternatively, you can [configure secrets in the Kubernetes console or set them up by using the Kubernetes CLI](https://docs.okd.io/latest/dev_guide/secrets.html#creating-secrets).
 
-### Running pipelines by using the pipelines dashboard webhook extension (Deprecated)
+### Running pipelines by using the Kabanero events operator webhooks
+
+You can create an organization webhook to automatically drive the pipelines based on events in the Github repo. Events such as commits, tagging, or pull requests can be set up to automatically trigger pipeline runs. Follow the instructions in the [Integrating events operator](integrating-events-operator) to setup your webhook.
```

Comment:
rewrite:
You can create an organization webhook to automatically drive the pipelines based on events in the GitHub repo. Events such as commits, tagging, or pull requests can be set to automatically trigger pipeline runs. Follow the instructions in the [Integrating events operator](../integrating-events-operator/) guide to set up your webhook.